core-feature-pack/common/pom.xml

```diff
@@ -212,6 +212,12 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.sshd</groupId>
+            <artifactId>sshd-common</artifactId>
+            <version>${version.org.apache.sshd}</version>
```

Comment:
FYI this version should not be in this pom.

Don't worry about fixing, I need to resolve a different merge conflict so will clean this at the same time.